deployment/ccip/changeset/add_lane.go

```diff
@@ -15,25 +17,103 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/router"
 )
 
+var _ deployment.ChangeSet[AddLanesConfig] = AddLanesWithTestRouter
+
 type InitialPrices struct {
 	LinkPrice *big.Int // USD to the power of 18 (e18) per LINK
 	WethPrice *big.Int // USD to the power of 18 (e18) per WETH
 	GasPrice  *big.Int // uint224 packed gas price in USD (112 for exec // 112 for da)
 }
 
+type Lane struct {
+	From uint64
+	To   uint64
```

Comment:
nit: Lets make an effort to standardize the source/dest jargon where possible to reduce this kinda unneeded naming variance

```suggestion
	SourceSelector uint64
	DestSelector   uint64
```